http-streaming/README.md

```diff
@@ -0,0 +1,297 @@
+║2024-03-17 20:02:59.000 06a080 QTR8112  Qatar          2024-03-17 20:02:58.000 8         46       7269                FALSE     230                ║
+║2024-03-17 20:02:59.000 4d223a RYR349   Malta          2024-03-17 20:02:58.000 10        47       10668               FALSE     206                ║
+║2024-03-17 20:02:59.000 4b17df SWR      Switzerland    2024-03-17 20:02:47.000 9         47       NULL                TRUE      0                  ║
+║2024-03-17 20:04:00.000 4b5d8e EUGEN18  Switzerland    2024-03-17 20:04:00.000 9         47       NULL                TRUE      0                  ║
+║2024-03-17 20:04:00.000 06a080 QTR8112  Qatar          2024-03-17 20:04:00.000 8         46       6751                FALSE     223                ║
+║2024-03-17 20:04:00.000 4d223a RYR349   Malta          2024-03-17 20:04:00.000 10        47       10668               FALSE     206                ║
+║2024-03-17 20:04:00.000 4b17df SWR      Switzerland    2024-03-17 20:03:56.000 9         47       NULL                TRUE      0                  ║
+║2024-03-17 20:04:59.000 4b5d8e EUGEN18  Switzerland    2024-03-17 20:04:59.000 9         47       NULL                TRUE      1                  ║
+║2024-03-17 20:04:59.000 a972ee GTI8069  United States  2024-03-17 20:04:59.000 7         48       9449                FALSE     252                ║
+║2024-03-17 20:04:59.000 06a080 QTR8112  Qatar          2024-03-17 20:04:58.000 8         46       6271                FALSE     219                ║
+║2024-03-17 20:04:59.000 4d223a RYR349   Malta          2024-03-17 20:04:58.000 10        47       10668               FALSE     207                ║
+║2024-03-17 20:04:59.000 4b17df SWR      Switzerland    2024-03-17 20:04:57.000 9         47       NULL                TRUE      0                  ║
+║2024-03-17 20:06:06.000 4b5d8e EUGEN18  Switzerland    2024-03-17 20:06:05.000 9         47       NULL                TRUE      0                  ║
+║2024-03-17 20:06:06.000 a972ee GTI8069  United States  2024-03-17 20:06:06.000 8         48       9449                FALSE     251                ║
+║2024-03-17 20:06:06.000 06a080 QTR8112  Qatar          2024-03-17 20:06:05.000 8         46       5799                FALSE     214                ║
+║                                                                                                                                                   ║
+╚═══════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╝
+```
+
+# OpenSky Network
+
+The demo in this repository connects to data provided by the [OpenSky Network](http://www.opensky-network.org) originally published in:
+
+>Bringing up OpenSky: A large-scale ADS-B sensor network for research
+Matthias Schäfer, Martin Strohmeier, Vincent Lenders, Ivan Martinovic, Matthias Wilhelm
+ACM/IEEE International Conference on Information Processing in Sensor Networks, April 2014
```

Comment:
another thought... at the end here a tear-down instruction for CC might be useful

Reply:
good call! this was an obversight